app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -446,13 +483,13 @@ const weatherIdMap = {
   620: <Snow duration={6} variance={1} count={20} />, // Light
   621: <Snow duration={4} variance={1} count={50} />, // Normal
   622: <Snow duration={2} variance={1} count={70} />, // Heavy
-  801: <Cloud duration={16} variance={1} count={2} />, // few cloud
-  802: <Cloud duration={16} variance={5} count={4} />, // scattered cloud
-  803: <Cloud duration={16} variance={10} count={8} />, // broken cloud
-  804: <Cloud duration={16} variance={15} count={16} />, // overcast cloud
+  801: () => <CloudWrapper variance={1} count={2} />, // few cloud
+  802: () => <CloudWrapper variance={5} count={4} />, // scattered cloud
+  803: () => <CloudWrapper variance={10} count={8} />, // broken cloud
+  804: () => <CloudWrapper variance={15} count={16} />, // overcast cloud
```

Comment:
This doesn't look like it passes the `windSpeed` property to the `CloudWrapper`.